src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -100,6 +106,145 @@ sealed trait Task {
 
 object Task {
 
+  import mesosphere.marathon.api.v2.json.Formats.{ TimestampFormat, PathIdFormat }
```

Comment:
I just moved stuff. I think that when we're talking about "moving stuff around in a separate commit because it's hard to review" means "our review tool is lacking". I didn't even know I needed to move stuff around until 80-90% through the work.

On that note, can we please stop using forward references? Eg if you use Task.Id, declare it first.